README.md

```diff
@@ -76,37 +75,97 @@ let dog = new Dog("Spot");
 dog.speak(); // Spot speaks! Spot says woof!
 ```
 In `Dog`s `constructor`, `super` is used as a `method`. Whereas, `Dog`'s `speak()`
-method used `super` as an `object`. When the `super` keyword is used as method, it
+method used `super` as an `object`. When the `super` keyword is used as a method, it
 calls the parent `class` `Pet` with the parameters passed to `Dog`. This is to ensure
 that `Dog` is an instance of `Pet`. When `super` is used as an `object` that refers to
-a `Pet` instance, it can calls methods of the parent `class` `Pet` explicitly.
+a `Pet` instance, it can call methods of the parent `class` `Pet` explicitly.
 
 ## Explain When `class` Inheritance Should Be Used in JavaScript
 
-JavaScript can be a bit confusing since it is a dynamic language that uses syntactical
-sugar to leverage `class` implementations and inheritance similar to classic programming
-languages. However, JavaScript's inheritance model is quite powerful and fairly trivial
-to build on top of like when using a classic programming language model. As the popular
-phrase goes: "With power comes great responsibility." It can be tempting to overuse these
-features. When creating "child" `class`es, we defer to _subtyping_ relation principles
-called _(strong) behavioral subtyping_ in programming language theory. 
-
-- You probably won't use inheritance often, but as your code bases get larger, you are more
+You probably won't use inheritance often, but as your code bases get larger, you are more
 likely to find a need for it. If you're creating a number of objects that have similar
 features, then creating a generic object type to contain all the shared functionality,
 and inheriting those features in more specialized objects can be convenient and useful.
 
-- When creating a parent-child `class` relation, it is _semantic_ rather than merely
-_syntactic_. Let's use `Vehicle` as a parent `class` with `Car` as a child
-`class` of `Vehicle` as an example. In _classical_ inheritance, it creates a _copy_ of the
-behavior from parent. However, in JavaScript when we create the object it does not copy the
-properties or behavior, it creates a _link_. `class`es should _never_ be extended unless it
-is for the sake of compatibility with newer JavaScript features. This is due to the way
-inheritance works on top of vanilla JavaScript code.
+It is recommended that `class`es should _never_ be extended unless it is for the sake of
+compatibility with newer JavaScript features, like building UI components frameworks in React
+and Angular. This is due to the way inheritance works on top of vanilla JavaScript code, and
+could create a multitude of unforseen headaches down the road. Always start with the simplest
+solution and progress to more complex solutions only as-needed.
+
+### JavaScript Inheritance is Not Like Other Languages
```

Comment:
Maybe take out the statement that JS is confusing since students may not even understand what dynamic language means anyway.

'JS is a dynamic language that uses...' Also, maybe if there is a wikipedia article that explains dynamic language, good to create a reference here